packages/graphql/src/types/app.d.ts

```diff
@@ -52,4 +57,15 @@ interface ChoicesQuestionResults {
   choices: Record<string, number>
 }
 
-type AllQuestionResults = ChoicesQuestionResults
+interface NumericalQuestionResults {
+  [x: string]: number
+}
+
+interface FreeTextQuestionResults {
+  [x: string]: number
+}
+
+type AllQuestionResults =
```

Comment:
these type unions allow TS to infer what kind of data we will have available if we, e.g., use options inside a switch statement which is based on the question.type